lib/scope.js

```diff
@@ -380,9 +390,35 @@ module.exports = {
       // 'none of the above' checkbox showif???
       `input[name="${ name_attr_B }"]`,  // checkbox
       `input[name="${ name_attr_R }"]`,  // checkbox alt
+
+      // simple index var case as described above
+      `button[name="${ base64_index }"][value="${ choice_name }"]`,  // button
```

Comment:
I think we can avoid all these repetition in selector clauses by setting a variable, e.g., `base_64_var_or_index` to either a variable name or an index variable. We have the ability to detect whether a given argument `var_name` is a variable name or an index name. So once we know which one it is, we can just use that.